server/services/visualization.js

```diff
@@ -0,0 +1,14 @@
+const Model = require('../models/visualization');
+
+module.exports = {
+    getVisualizationByTitle: async function getVisualizationByTitle(title) {
```

Comment:
Same as above, we just want our services to be a collection of async functions that throw errors that we define in a file called errors.js